hbase-server/src/main/java/org/apache/hadoop/hbase/namequeues/NamedQueueServiceChore.java

```diff
@@ -27,11 +28,12 @@
  * Chore to insert multiple accumulated slow/large logs to hbase:slowlog system table
  */
 @InterfaceAudience.Private
-public class SlowLogTableOpsChore extends ScheduledChore {
```

Comment:
Earlier SlowLogTableOpsChore was used to persist only slow logs to hbase:slowlog table. In this feature, we have a need to persist WAL events to wal event tracker table. Instead of creating a new chore for Wal event tracker table, we thought to re-purpose SlowLogTableOpsChore and generalized it to persist any events based on configured QueueService.